images/router/haproxy/conf/error-page-404.http

```diff
@@ -0,0 +1,148 @@
+HTTP/1.0  404 Error, Page Not Found, File Not Found, or Server Not Found

```

Comment:
I'm not sure what the restrictions on the reason phrase are, but reason phrases are usually much more terse:
```suggestion
HTTP/1.0 404 Not Found
```